rpc/js/src/dial.d.ts

```diff
@@ -15,6 +15,7 @@ export interface DialWebRTCOptions {
   signalingExternalAuthToEntity?: string;
   signalingCredentials?: Credentials;
   signalingAccessToken?: string;
+  priority?: number;
```

Comment:
I don't think we should expose priority directly - we should have an `additionalSDPFields` or something (that is an object with key-value pairs) so that if we want to add future fields we don't have to make additional updates.